src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url, header) {
```

Comment:
You are passing 'header' as a parameter to the constructor but not using it. Instead, you're creating a new instance of the Header class. If the intention is to pass an existing instance of Header, use 'this.header = header', otherwise, no need to pass 'header' as a parameter. Remove

Reply:
done